### PR TITLE
"Spell-Jacking" mitigation ~ prevent sensitive data leak …

### PR DESCRIPTION
**Pull request by dlehammer**
… from spell checker.

Background: [Chrome & Edge Enhanced Spellcheck Features Expose PII, Even Your Passwords](https://www.otto-js.com/news/article/chrome-and-edge-enhanced-spellcheck-features-expose-pii-even-your-passwords)

### CODE REVIEW

File: templates/common/forgot-password.html [hunk 1]
```diff
@@ -68,7 +68,7 @@ <h1 class="h4 text-gray-900 mb-4">Forgot Your Password?</h1>
                                         class="user-custom">
                                         <div class="form-group">
                                             <input type="text" class="form-control form-control-user-custom"
-                                                id="inputUsername" name="username" placeholder="Your username" required>
+                                                id="inputUsername" name="username" placeholder="Your username"spellcheck="false" required>
```
Comment (drakkan) [inline templates/common/forgot-password.html:71]: a space seems missing here

**Comment by drakkan** [comment]
It seems a good addition, thanks. I'll better review your changes the next week. In the meantime wait for the CI to complete and then rebase on current main branch to avoid merge conflicts. Thanks

File: templates/common/forgot-password.html (hunk 1 above)
Comment (dlehammer) [reply templates/common/forgot-password.html]: Nice catch :pray:

**Comment by dlehammer** [comment]
Thanks for the pointers, I've added the missing space + rebased.

Looking forward to your insights :)

**Comment by drakkan** [comment]
There are some minor things I'll fix myself in another commit. Thanks

**Pull request merged**